BlogAPI.Persistence/Models/BlogPost.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace BlogAPI.Persistence.Entities
+{
+    public class BlogPost
+    {
+        public int Id { get; set; }
+        public string Title { get; set; }
+        public string Author { get; set; }
+        public string Content { get; set; }
+        public DateTime CreationDate { get; set; }
+        public ICollection<Comment> Comments { get; set; } = new List<Comment>();
+
+        public ICollection<string> Tags { get; set; } = new List<string>();
```

Comment:
Why are all of the properties public? You should [encapsulate ](https://www.geeksforgeeks.org/c-sharp-encapsulation/) them. It's fine to have the public getters, but setters should be private.

Reply:
I agree, most of these properties should be set from some sort of constructor.